test/helpers/utils.js

```diff
@@ -53,6 +52,29 @@ export function getSigs(wallets, votedata) {
     .filter(d => d)
 }
 
+export function getSigsWithVotes(wallets, data, sigPrefix, maxYesVotes) {
+  wallets.sort((w1, w2) => {
```

Comment:
Also make a copy of the wallets, last i time i was wondering why my input array was re-ordered.

Reply:
done